plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -16,112 +18,151 @@ Even still, there are times when creating a custom table can be more effective.
 
-The `wpdb` class is used to manage a database connection while also providing helpful methods to perform queries and retrieve data from the connected database in a structured way. In PHP, WordPress automatically sets the `$wpdb` global variable to an instance of this class by using the `DB_HOST`, `DB_NAME`, `DB_USER`, and `DB_PASSWORD` constants defined in your `wp-config.php` file.
+During PHP execution, WordPress creates a `$wpdb` global object variable as an instance of this class by using the `DB_HOST`, `DB_NAME`, `DB_USER`, and `DB_PASSWORD` constants defined in your `wp-config.php` file.
 
-You can access it in PHP like this:
+Because the `$wpdb` object exists in the global namespace, you can access it in PHP like this:
 
 ```php
 global $wpdb;
 ```
 
-The `$wpdb` instance also contains crucial information about the WordPress database which we'll need to properly create and manage our own custom tables. Let's see how as we create our first one.
+The `$wpdb` instance also contains crucial information about the WordPress database which you'll need to properly create and manage your own custom tables. Let's see this in action as we create our first one.
 
 ## Creating a Custom Table
 
-To start, register an activation hook for your plugin to create the custom table. Using this hook ensures the table immediately exists once your plugin is activated so that any following code can safely query against it.
+To start, create a new plugin to store the custom table code. 
+
+```php
+<?php
+/**
+ * Plugin Name: Form submissions
+ * Version:     1.0.0
+ * Author:      Learn WordPress
+ * Author URI:  https: //learn.wordpress.org/
+ *
+ * @package wp-learn-form-submissions
+ */
+ ```
 
-When choosing a name for the custom table, be sure to use the configured table prefix by prepending the table name with the `$wpdb->prefix` variable. Also, use `$wpdb->get_charset_collate()` to ensure the custom table's collation matches the rest of the tables in WordPress for optimum data integrity, compatibility, and query performance.
+Then, register an activation hook for your plugin to create the custom table. Using this hook ensures the table immediately exists once your plugin is activated so that any following code can safely query against it.
 
-To create a custom table, use the `dbDelta()` function included in WordPress. This function is powerful because it can handle both table creation and future updates to the table's structure.
+When choosing a name for the custom table, be sure to use the configured table prefix by prepending the table name with the `$wpdb->prefix` object property. Also, use the `$wpdb->get_charset_collate()` method to ensure the custom table's collation matches the rest of the tables in WordPress for optimum data integrity, compatibility, and query performance.
```

Comment:
As we assume the table prefix to be `wp_` later in the lesson, I do want to clarify this early on to say:

>When choosing a name for the custom table, be sure to use the configured table prefix by prepending the table name with the `$wpdb->prefix` object property. The default table prefix is `wp_` for single sites and `wp_{$blogid}_` (eg. `wp_2_`) for subsites within [WordPress multisites](https://developer.wordpress.org/advanced-administration/multisite/).
>
>Also, use the `$wpdb->get_charset_collate()` method to ensure the custom table's collation matches the rest of the tables in WordPress for optimum data integrity, compatibility, and query performance.

I just don't want a developer to be confused if they try to run a query against their table or go looking for their custom table, not realizing it can vary from the typical `wp_`. I've seen some hosting providers impose a custom prefix for security reasons, for example.